### PR TITLE
Bump log4j to 2.17.0

### PR DESCRIPTION
**Pull request by kkocel**
## Motivation and Context
Move to the most recent version that is known to be free of vulnerabilites.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have read the **README** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

## License
- [x] By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

[issues]: https://github.com/alexa/alexa-skills-kit-sdk-for-java/issues
[license]: http://aws.amazon.com/apache2.0/
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

File: samples/colorpicker/pom.xml [hunk 1]
```diff
@@ -36,7 +36,12 @@
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-slf4j-impl</artifactId>
-      <version>2.15.0</version>
+      <version>2.17.0</version>
+    </dependency>
+    <dependency>
```
Comment (doiron) [inline samples/colorpicker/pom.xml]: this appears to be a duplicate of the above dependency.   Could you remove one?

File: samples/helloworld/pom.xml [hunk 2]
```diff
@@ -36,7 +36,12 @@
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-slf4j-impl</artifactId>
-      <version>2.15.0</version>
+      <version>2.17.0</version>
+    </dependency>
+    <dependency>
```
Comment (doiron) [inline samples/helloworld/pom.xml]: same here could you remove one of the org.apache.logging.log4j  dependency declaration?
Thx

File: samples/colorpicker/pom.xml (hunk 1 above)
Comment (kkocel) [reply samples/colorpicker/pom.xml]: removed

File: samples/helloworld/pom.xml (hunk 2 above)
Comment (kkocel) [reply samples/helloworld/pom.xml]: removed

**Comment by doiron** [comment]
just updated the 2.0.x branch with these changes. thanks for the PR!

**Pull request closed without merging**